### PR TITLE
refactoring navbar sticky

### PR DESCRIPTION
**Pull request by rupal-2505**
### **Issue**
#235 

### **Current status**
Currently , when user scroll-down the RDS website, the navbar is scrolled down . 

Screenshot (75)

### **What is the change?**
Navbar is fixed .Will not scroll with the page.

Screenshot (84)

### CODE REVIEW

**Review by RitikJaiswal75 — COMMENTED** [review]
Please put a proper video of the changes

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "license": "ISC",
   "devDependencies": {
     "pre-commit": "^1.2.2",
-    "prettier": "2.2.1"
+    "prettier": "2.7.1"
```
Comment (RitikJaiswal75) [inline package.json:15]: why did this change is it related to your PR?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (RitikJaiswal75) [inline yarn.lock:1]: Why are you changing the yarn lock?

File: index.html [hunk 3]
```diff
@@ -32,90 +32,89 @@
   </head>
 
   <body>
-    <div class="first-screen">
```
Comment (RitikJaiswal75) [inline index.html:35]: Was removing this Div Really required?

**Review by RitikJaiswal75 — COMMENTED** [review]
@rupal-2505 In many places i can see that you have changed lot of styling related to width margins and padding, I understand the top margin part where you want the navbar to look sticky to the top but other places do not really make sense for this PR.

File: yarn.lock (hunk 2 above)
Comment (RitikJaiswal75) [inline yarn.lock:1]: Do not change the yarn lock

File: package.json (hunk 1 above)
Comment (RitikJaiswal75) [inline package.json:15]: is it really required by your PR?

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (rupal-2505) [reply index.html:35]: yeah, it was required as the navbar and main content container  was in one container so have to separate them.

**Comment by rupal-2505** [comment]
> @rupal-2505 In many places i can see that you have changed lot of styling related to width margins and padding, I understand the top margin part where you want the navbar to look sticky to the top but other places do not really make sense for this PR.

File: package.json (hunk 1 above)
Comment (rupal-2505) [reply package.json:15]: no, it was not required but to format the code i used it.

**Comment by rupal-2505** [comment]
> Please put a proper video of the changes

@RitikJaiswal75 Added video for reference.

File: css/index.css [hunk 4]
```diff
@@ -23,6 +23,12 @@ body {
   font-family: 'raleway';
   background-color: var(--color-bg);
 }
+header {
+  position: sticky;
+  top: 0;
+  background-color: #1d1283;
```
Comment (RitikJaiswal75) [inline css/index.css:29]: please use a css variable here

File: css/navbar-linked-pages.css [hunk 5]
```diff
@@ -1,5 +1,8 @@
 nav {
   font-weight: 700;
+  position: sticky;
+  top: 0;
+  background-color: #1d1283;
```
Comment (RitikJaiswal75) [inline css/navbar-linked-pages.css:5]: please use css variable

File: css/navbar.css [hunk 6]
```diff
@@ -1,6 +1,9 @@
 nav {
   font-family: 'roboto';
   font-weight: 700;
+  position: sticky;
+  top: 0;
+  background-color: #1d1283;
```
Comment (RitikJaiswal75) [inline css/navbar.css:6]: css variable please

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**